react/tic-tac-toe/src/app/screens/Game/components/Board/index.js

```diff
@@ -1,37 +1,48 @@
-import React, { Component } from 'react';
+import React from 'react';
+import PropTypes from 'prop-types';
 
 import Square from '../Square';
 
 import styles from './styles.module.scss';
 
-class Board extends Component {
-  renderSquare() {
-    return <Square />;
-  }
-
-  render() {
-    const status = 'Next player: X';
-    return (
-      <div>
-        <div className={styles.status}>{status}</div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(0)}
-          {this.renderSquare(1)}
-          {this.renderSquare(2)}
-        </div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(3)}
-          {this.renderSquare(4)}
-          {this.renderSquare(5)}
-        </div>
-        <div className={styles.boardRow}>
-          {this.renderSquare(6)}
-          {this.renderSquare(7)}
-          {this.renderSquare(8)}
-        </div>
+function Board ({ status, squares, handleClick }) {
+  return (
+    <div>
```

Comment:
Replace `<div>` to `<Fragment>` and in the ultimate `</div>` replace to `</Fragment>`. You can read of this [here](https://reactjs.org/docs/fragments.html)

For thew use Fragment you can `import { Fragment } from 'react';`